libs/feature/dataviz/src/lib/chart-view/chart-view.component.ts

```diff
@@ -197,6 +202,12 @@ export class ChartViewComponent {
         .groupBy(['distinct', xProp])
         .aggregate(fieldAgg)
         .read()
+        .then((data) => {
+          if (this.maxFeatureCount && data.length > this.maxFeatureCount) {
+            this.handleError('maxFeature')
+          }
+          return data
+        })
```

Comment:
This check doesn't make sense when we're aggregating, because the resulting amount of items after aggregation is most likely lower than the total count; instead, this check should happen earlier when determining whether a WFS link is appropriate for the chart view